src/store/reducers/node/utils.ts

```diff
@@ -0,0 +1,16 @@
+import type {TEvSystemStateResponse} from '../../../types/api/systemState';
+import type {PreparedNode} from './types';
+
+export const prepareNodeData = (data: TEvSystemStateResponse): PreparedNode => {
```

Comment:
Although is looks similar to other nodes preparation functions, it's different endpoint - `/viewer/json/sysinfo`, so there should be different handler